app/models/loan.rb

```diff
@@ -11,14 +11,10 @@ class Loan < ActiveRecord::Base
   DURATION = 30
   MAX_RENEWALS = 2
 
-  def return_loan
```

Comment:
removing this method has caused a test failure in:
 ./spec/models/loan_spec
and
./spec/requests/loan_pages_spec.rb

Since this is handled by the controller now, I think we can just remove the model test. For the request test, we can just add the returned day into the call to create on line 58 (and remove the call the the old method).